tools/nodejs/test/tokenize.test.ts

```diff
@@ -0,0 +1,57 @@
+import * as assert from 'assert';
+import * as duckdb from '..';
+
+describe('tokenize', function () {
```

Comment:
Can you add a test for handing invalid syntax?

Reply:
Ah, good point, will do.

Reply:
I added one test, but of course there are many variations on invalid syntax. The scanner doesn't throw on invalid syntax, though - it'll just mark tokens as different types.